eslzArm/subscriptionTemplates/nvahubspoke-connectivity.json

```
@@ -123,9 +127,11 @@
         "rgName": "[concat(parameters('topLevelManagementGroupPrefix'), '-vnethub-', parameters('location'))]",
         "hubName": "[concat(parameters('topLevelManagementGroupPrefix'), '-hub-', parameters('location'))]",
         "azVpnGwIpName": "[concat(variables('vpngwname'), '-pip')]",
+        "azVpnGwAAIpName": "[concat(variables('vpngwname'), '-aa-pip')]",
```

```suggestion
        "azVpnGwAAIpName": "[concat(variables('vpngwname'), '-pip-002')]",
```

Making it more aligned to CAF recommendations by adding an instance to the end instead of AA, as technically both IPs are for A/A.

> Not renaming the other IP to avoid a breaking change for anyone who redeploys the portal experience